src/CpmsCommon/AbstractInputFilterFactory.php

```diff
@@ -63,18 +63,20 @@ public function getClassName($name, ContainerInterface $serviceLocator)
             $namespaceParts[$i] = ucfirst($namespaceParts[$i]);
         }
 
-        if (count($namespaceParts) > 0 and in_array($namespaceParts[0], $this->versionedNamespaces)) {
-
-            $config         = $serviceLocator->get('config');
+        if (in_array($namespaceParts[0], $this->versionedNamespaces)) {
```

Comment:
I believe the behaviour here is now changed since we dont check if var is more than 0.

Reply:
It was complaining that the `count($namespaceParts)` would never be zero, I checked it in the PHP REPL and it will always have a length of at least 1, because the explode will give an empty string, even if it's passed an empty string. So that `count` test is never false. 

Based on that, I think we're ok.